internal/server/data/migrations.go

```diff
@@ -883,3 +884,14 @@ func addDestinationKind() *migrator.Migration {
 		},
 	}
 }
+
+func addAllowedDomainsToProvidersTable() *migrator.Migration {
+	return &migrator.Migration{
+		ID: "2022-11-03T13:00",
+		Migrate: func(tx migrator.DB) error {
+			stmt := `ALTER TABLE providers ADD COLUMN IF NOT EXISTS allowed_domains text DEFAULT ''`
```

Comment:
could consider making this a jsonb field, though that would not work with `CommaSeparatedStrings`.. so it would need some extra work.

Reply:
Let's do this in a separate change, not allowing commas in the domains is ok for now, jsonb would be nice for lookup though